libs/gl-client/src/signer/mod.rs

```diff
@@ -225,6 +225,7 @@ impl Signer {
                 if r.timestamp != 0 {
                     data.put_u64(r.timestamp);
                 }
+                data.put(&r.rune[..]);
```

Comment:
This is a breaking change: changing the way we verify signatures means that we can't mix pre-change clients and post-change signers and vice-versa. This is not an issue in this case because we have a rather tight coupling still, but better keep this in mind for when we have keyless clients as they won't upgrade in lockstep.